front/src/plugins/kakao-map.js

```diff
@@ -153,24 +169,31 @@ export const KakaoMap = (() => {
     return new kakao.maps.MarkerImage(src, size, options);
   };
 
-  const _createMarker = (position, image) => {
-    return new kakao.maps.Marker({
+  const _createMarker = (map, position, image) => {
+    const newMarker = new kakao.maps.Marker({
+      map: map,
       position: position,
       image: image,
     });
+    return newMarker;
   };
 
-  const setMarker = (location, img) => {
+  const setMarker = (location) => {
     if (!map || !location) {
       return;
     }
-    const markerSize = new kakao.maps.Size(36, 36);
-    const markerImage = _createMarkerImage(img, markerSize);
+    const kakaoLocation = _createKakaoLocation(location);
+    return _createMarker(map, kakaoLocation, null);
```

Comment:
```suggestion
    return _createMarker(map, kakaoLocation);
```

null을 굳이 안넣어도 되긴 하는데... 흠.... 괜찮은 방법일지는 또 모르겠네요.

Reply:
흠 이문제는 팀벤션으로 가져가도 될 것 같아요.
개인적으로 전 명시적으로 null임을 표시하는 게 좋다고 생각하지만 (중간에 있는 kakaoLocation이 null일 경우도 있으니까), 또 프론트엔드 영역에선 보통 어떻게 처리하는지 궁금하네요.